common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -236,12 +246,14 @@
             };
 
             DiscussionThreadListView.prototype.render = function() {
-                var self = this;
-                this.timer = 0;
-                this.$el.html(this.template({
+                var self = this, $elem;
```

Comment:
Nit: new JS style guide (which as I said above hasn't been turned on here yet so it's not totally fair for me to review things against it...) says to [not do initialization and instantiation on the same line](https://github.com/edx/eslint-config-edx#one-var-declaration-per-line). Since you assign a value to `$elem` on the next line, may as well just:

```javascript
var self = this,
    $elem = this.template({
        ...
    });
```